packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-subgraph/index.esm.t

```diff
@@ -0,0 +1,43 @@
+---
+to: "<%= rootDir %>/<%= h.changeCase.param(name) %>/index.ts"
+force: true
+---
+import { Subgraph } from "@powerhousedao/reactor-api";
+import { Knex } from "knex";
```

Comment:
Knex is only used as a type here right?
To avoid adding it as a dependency in the boilerplate we could export the type from "reactor-api".
We could even rename it to Db:
`export { Knex as Db } from "knex";`